neqo-transport/tests/server.rs

```diff
@@ -18,7 +18,7 @@ use neqo_crypto::{
 use neqo_transport::{
     server::{ActiveConnectionRef, Server},
     Connection, ConnectionError, Error, FixedConnectionIdManager, Output, State, StreamType,
-    QUIC_VERSION,
+    DEFAULT_QUIC_VERSION,
```

Comment:
We'll need tests with both versions of client.  Though it's probably enough to just have a single -27 test.

Reply:
covered by issue #692 I think.